src/Adapter/AbstractAdapter.php

```diff
@@ -6,17 +6,17 @@
 
 abstract class AbstractAdapter implements ValidatableAdapterInterface
 {
-    /** @var mixed */
+    /** @var mixed|null */
```

Comment:
Note: `mixed` contains `null`

Reply:
ref https://3v4l.org/HM6vQ

Reply:
Yes! It's a weird habit I've got into. Considering it's not a valid union, I've replaced with just `mixed` 👍